src/TMap.cpp

```diff
@@ -2685,8 +2685,7 @@ void TMap::slot_replyFinished(QNetworkReply* reply)
     if (reply->error() != QNetworkReply::NoError) {
```

Comment:
I think you can combine these two `if`s into a single `&&`ed one:
```cpp
if (reply->error() != QNetworkReply::NoError && reply->error() != QNetworkReply::OperationCanceledError) {
    // Don't report on any errors here as we've already done so in slot_downloadError(...) previously.
    // Just do the cleanup:
    cleanup();
    return;
}    
```
and omit the `// else was QNetworkReply::OperationCanceledError ...` comment as well as it doesn't really fit here since all the relevant error messages are handled in the `slot_downloadError(...)`.

Thanks for working out this fix (in what was probably my code anyhow... :blush:) - I hope the responsible party will get the bounty out to you as soon as we get this into the code...